Darwin/Examples/ToDoSync/ToDoSync/AppDelegate.swift

```diff
@@ -0,0 +1,38 @@
+import UIKit
+import StitchCore
+import StitchRemoteMongoDBService
+
+private let todoListsDatabase = "todo"
+private let todoItemsCollection = "items"
+private let todoListsCollection = "lists"
+private let todoIndexSwapsCollection = "index_swaps"
+
+let stitch = try! Stitch.initializeAppClient(withClientAppID: "todo-cqenc")
```

Comment:
[nit q] are we fine hardcoding this stitch client app id? we do it in android too but it means anyone who looks at this repo and can see & edit the todo items in this app

Reply:
I don't see an issue with that at the moment. I agree, it's odd, but can be removed down the line.